lang_qc/db/helper/wells.py

```diff
@@ -167,16 +178,23 @@ def create(self) -> PacBioPagedWells:
 
         The `PacBioWell` objects in `wells` attribute of the returned object
         are sorted in a way appropriate for the requested `qc_flow_status`.
-        For non-inbox requests the wells with most recently assigned QC states
-        come first. For inbox requests the wells with least recent 'run completed'
-        timestamp are listed first. The query for the inbox wells looks at runs
-        which completed within the last four weeks.
+        For the 'in progress' and 'on hold' requests the wells with most recently
+        assigned QC states come first. For inbox requests the wells with least
+        recent 'run completed' timestamp are listed first. The query for the
+        inbox wells looks at runs which completed within the last four weeks.
+        For the 'aborted' and 'unknown' queries the wells are sorted in the
+        run name alphabetical order.
         """
 
         wells = []
         if self.qc_flow_status == QcFlowStatusEnum.INBOX:
             recent_wells = WellWh(session=self.mlwh_session).recent_completed_wells()
             wells = self._recent_inbox_wells(recent_wells)
+        elif self.qc_flow_status in [
+            QcFlowStatusEnum.ABORTED,
+            QcFlowStatusEnum.UNKNOWN,
+        ]:
+            wells = self._aborted_and_unknown_wells()
```

Comment:
Aha, perhaps you could change L191 to match this `self` invocation? It seems to silly to instantiate a new one in this context.

Reply:
On line 91 an object of a different class is instantiated. This object (self) does not inherit from it. Might be it should.

Reply:
Now it does, see https://github.com/wtsi-npg/npg_langqc/pull/110/commits/5f053f9f77e07b053e67230247cdfaab8c604c24